### ISSUE TITLE
When I Stake to private PoS network.

### ISSUE THREAD

**Issue by KP-Universe** [wealdtech/ethereal#51]
I have a question sir.

I'm trying to deposit to stake my network which is custom network(https://docs.prylabs.network/docs/advanced/proof-of-stake-devnet)

And I'm following some steps to stake(https://docs.prylabs.network/docs/install/install-with-script)

I got geth client and beacon-client, and I changed all of the holesky config from prysm, staking-deposit-cli, ethereal

It's my steps.
1. create deposi data : ./deposit.sh new-mnemonic --num_validators=4 --mnemonic_language=english --chain=holesky
2. create validator wallet : ./validator wallet create --keymanager-kind imported --wallet-dir validator_keys --holesky --wallet-password-file validator_keys/password.txt
3. validator account import : ./validator accounts import --keys-dir validator_keys --holesky
4. check validator list : ./validator accounts list --holesky --wallet-dir=/home/hun/devnet/validator_keys
5. deposit to deposit contract on my chain(usually do this on Launchpad) : ./ethereal beacon deposit --connection=gethdata/geth.ipc --data=/home/hun/devnet/validator_keys/deposit_data-1706365759.json --from=d088a8031953a777b065976b7cc3a0d381c097cc --passphrase=142536 --eth2network=holesky
6. I checked 4 transaction was completed : 
![image](https://github.com/wealdtech/ethereal/assets/155970903/65d6882a-bc6b-4284-9ee7-2e6604a0b44c)
![image](https://github.com/wealdtech/ethereal/assets/155970903/6a09ca7a-690c-4ada-9c76-2e07acd9714f)
7. start validator : ./validator --wallet-dir=/home/hun/devnet/validator_keys --suggested-fee-recipient=0xd088a8031953a777b065976b7cc3a0d381c097cc --datadir validatordata --beacon-rpc-provider 127.0.0.1:4000 --chain-config-file config.yml
![image](https://github.com/wealdtech/ethereal/assets/155970903/12f30dee-0240-4579-80b0-9f8bb774b968)

But as you can see the picture that I uploaded they just keep validating.... and stop not working
The step number 5 is correct timing to do that? or should I do something more?

**Comment by mcdee** [comment]
Your problem is likely in step 1, where deposit data is created using the holesky genesis fork version.  However, this is not an ethereal issue so I'm closing this.

**Comment by KP-Universe** [comment]
> Your problem is likely in step 1, where deposit data is created using the holesky genesis fork version. However, this is not an ethereal issue so I'm closing this.

Thank you so much for answering my questions, and I am truly sorry. This issue was my fault. I made a mistake by omitting a necessary option in the final step. Everything worked perfectly when I changed the option in the last step, and I was able to confirm that I'm acting as a validator and receiving fees. Thank you again for responding to my naive questions.

P.S.: I changed all the Holesky related settings in every project to my network settings.
